ssp

```diff
@@ -39,66 +39,66 @@ use Getopt::Long;
 our $VERSION = '4.99.170';
 
 # Global variables that alter application runtime
-our $OPT_TIMEOUT;      # How long to wait for system commands to finish executing
+our $opt_timeout;    # How long to wait for system commands to finish executing
```

Comment:
De-capitalizing the globals makes it more difficult to differentiate a locally scoped variable and a global variable when writing and reviewing SSP code, so I do not agree with these changes as it currently stands.  Is there a more suitable way for the globals to stand out visibly?  Capitalization has been working well up to this point.

Reply:
http://www.catb.org/jargon/html/Y/yak-shaving.html

I didn't think about visibility, because looking at a function for variables that are used and not declared seemed intuitive enough for me. This isn't necessary though; I'll pull it back out.